### PR TITLE
added license and readme

### PR DESCRIPTION
**Pull request by kxwu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+### LaunchDarkly Sample Javascript Application  ###
+We've built a simple console application that demonstrates how LaunchDarkly's SDK works.  Below, you'll find the basic build procedure, but for more comprehensive instructions, you can visit your [Quickstart page](https://app.launchdarkly.com/quickstart#/).
+##### Build instructions  #####
+1. Copy your feature flag key from your LaunchDarkly dashboard into the snippet in `index.html`
```
Comment (jkodumal) [inline README.md]: You also have to copy a JavaScript snippet into `index.html`. Take a look at the Quickstart instructions for more accurate text.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+### LaunchDarkly Sample Javascript Application  ###
+We've built a simple console application that demonstrates how LaunchDarkly's SDK works.  Below, you'll find the basic build procedure, but for more comprehensive instructions, you can visit your [Quickstart page](https://app.launchdarkly.com/quickstart#/).
+##### Build instructions  #####
+1. Drop the snippet from the [Tutorial](https://app.launchdarkly.com/quickstart#/tutorial) section into the `<head>` section of `index.html`
+2. Copy your feature flag key from your LaunchDarkly dashboard into the snippet in `index.html`
```
Comment (jkodumal) [inline README.md]: This should not be referred to as a snippet. Just "into `index.html`" should be fine.

Comment (kxwu) [inline README.md]: Oh I missed that. I'll get on it

**Comment by jkodumal** [comment]
lgtm

**Pull request merged**